packages/toast/src/Toast.story.tsx

```diff
@@ -10,24 +12,38 @@ export default {
     body: 'Exercitation incididunt ea proident velit mollit',
     open: true,
     variant: Variant.Note,
+    darkMode: false,
   },
   argTypes: {
     className: {
       control: 'string',
     },
-    progress: {
-      control: 'range',
-      min: 0,
-      max: 1,
-      step: 0.01,
-      initialValue: 0,
-    },
+    progress: { control: { type: 'range', min: 0, max: 1, step: 0.1 } },
```

Comment:
I think there's a way to render this conditionally based on the Variant prop (I think we do this in Combobox?)

Reply:
There are also some JSON controls that I think we should change 
![Screen Shot 2022-09-02 at 2 41 15 PM](https://user-images.githubusercontent.com/2414030/188217626-9f8769dd-46be-4f05-b25a-db7412d9c53e.png)